package.json

```diff
@@ -7,9 +7,11 @@
     "dev:client": "NODE_ENV=development webpack -w",
     "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
     "prod:build": "NODE_ENV=production webpack -p",
-    "format": "prettier --write 'src/**/*.js'",
-    "lint": "eslint 'src/**/*.js'",
-    "test": "npm run lint",
+    "format": "prettier --write '{src,test}/**/*.js'",
+    "lint": "eslint '{src,test}/**/*.js'",
+    "pretest": "npm run lint",
+    "test": "mocha '{src,test}/**/*.test.js'",
+    "test:watch": "chokidar '**' --silent -c 'npm test'",
```

Comment:
I'm not sure if you realized but I installed `nodemon` previously which wraps chokidar and some other niceties and does basically the same with a nice API. For example you can trigger restarts by typing `rs` in the watching process without having to touch a file.

I don't mind switching to raw chokidar but I strongly believe that we should be only using one file watching utility.

This command with nodemon I believe would be something like: 

    nodemon -w src -w test --exec "npm test"

Let me know what you think.